app/views/service_providers/_form.html.erb

```diff
@@ -280,7 +280,7 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
                  input_html: { class: 'usa-input' },
                  label_html: { class: 'usa-input-required'},
                  label: 'Failure to Proof URL',
-                 hint: I18n.t('service_provider_form.failure_to_proof_url'),
+                 hint: I18n.t('service_provider_form.failure_to_proof_url').html_safe,
```

Comment:
I think we should prefer renaming the key of the YAML content to `failure_to_proof_url_html`, which should automatically take care of escaping. Even better would be to separate out the HTML/content, but that'd be more involved.

Reply:
Tried changing the name as suggested and dropping the html_safe, and that doesn't seem to do it automatically. I'll dig into it a bit to see whats going on.

Reply:
https://github.com/18F/identity-idp/blob/67df16d/config/locales/idv/en.yml#L264 is an example on the IDP side

Reply:
I think it's as straightforward as using `t()`

Reply:
Oh I missed that this was merged

https://github.com/18F/identity-dashboard/compare/mitchellhenke/translation-helper shows the changes needed